backend/stakepoold/ntfnhandlers.go

```diff
@@ -10,12 +10,33 @@ import (
 // Define notification handlers
 func getNodeNtfnHandlers(ctx *appContext, connCfg *dcrrpcclient.ConnConfig) *dcrrpcclient.NotificationHandlers {
 	return &dcrrpcclient.NotificationHandlers{
-		OnWinningTickets: func(blockHash *chainhash.Hash, blockHeight int64,
-			winningTickets []*chainhash.Hash) {
+		OnNewTickets: func(blockHash *chainhash.Hash, blockHeight int64, stakeDifficulty int64, tickets []*chainhash.Hash) {
+			nt := NewTicketsForBlock{
+				blockHash:   blockHash,
+				blockHeight: blockHeight,
+				newTickets:  tickets,
+			}
+			ctx.newTicketsChan <- nt
+		},
+		OnSpentAndMissedTickets: func(blockHash *chainhash.Hash, blockHeight int64, stakeDifficulty int64, tickets map[chainhash.Hash]bool) {
+			ticketsFixed := make(map[*chainhash.Hash]bool)
```

Comment:
By no means am I certain there's an issue, but two things feel funny. First is that memory leaks become much more likely with pointers, especially in the case of a map where you have buckets that need to be freed an so forth.  Second, how does a look up work?  Say you come in with another tx hash and want to see if it's in the table, can you get to it?  Hasn't the map hashed the pointer as a key, not the actual `chainhash.Hash`?